README.md

```diff
@@ -9,7 +9,7 @@ A wrapper for the [pigpio C library](https://github.com/joan2937/pigpio) to
 enable fast GPIO, PWM, servo control, state change notification and interrupt
 handling with **Node.js** on the Raspberry Pi Zero, 1, 2, 3 or 4.
 
-pigpio supports Node.js versions 10, 12, 14, 15 and 16.
+pigpio supports Node.js versions 10, 12, 14, 15 and 16 (but version 18 seems to work as well).
```

Comment:
I don't like the idea of adding text that says that something might work. In reality, I would also expect pigpio to function with Node.js v18, but I haven't tested it extensively yet.

Reply:
agreed. Do you have a recommendation, which tests I should run (without requiring too much external hardware) in order to verify my assumption?